monasca-python/Dockerfile

```diff
@@ -3,5 +3,12 @@ ARG ALPINE_VERSION="3.6"
 
 FROM python:${PYTHON_VERSION}-alpine${ALPINE_VERSION}
 
-COPY build.sh /
+COPY build.sh stack-fix.c /
 RUN chmod +x /build.sh
+
+RUN apk add --no-cache ca-certificates tini && \
+  apk add --no-cache --virtual build-dep musl-dev linux-headers git make g++ linux-headers && \
+  gcc -shared -fPIC /stack-fix.c -o /stack-fix.so && \
+  apk del build-dep
+
+ENTRYPOINT ["LD_PRELOAD=/stack-fix.so", "/sbin/tini", "--"]
```

Comment:
I think we'd need to be running in a shell for `VAR=value` to work (which `ENTRYPOINT` is not). So we probably need:

```
ENTRYPOINT ["/sbin/tini", "--"]
CMD ["LD_PRELOAD=/stack-fix.so", "python", "foo.py"]
```

The LD_PRELOAD override will probably need to be done inside each container since it needs to be in the `CMD` directive or some startup script. We could use `ENV LD_PRELOAD=/stack-fix.so` but I'd almost prefer that not to be user-overridable...